src/index.html

```diff
@@ -17,6 +20,12 @@
     />
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="hand">
+        <div class="hand_minutes"></div>
+        <div class="hand_seconds"></div>
```

Comment:
The class 'hand_seconds' should follow BEM naming convention. If it's an element of 'hand', it should be 'hand__seconds'.